### PR TITLE
feat: create Map setting page for plateau view 

### PR DESCRIPTION
**Pull request by nina992**
### Overview
The main purpose is to create plugins that shows plateau view functionalities, and one of them map setting 
please refer to this https://github.com/eukarya-inc/reearth-plateauview/issues/25
  
### What I've done
1. create the map setting page 
### Screenshot
image

### CODE REVIEW

File: plugin/web/components/sidebar/tabs/MapSettingTab.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    key: "3",
+    title: "GSI Maps (light color)",
+    icon: bgmap_gsi,
+  },
+  {
+    key: "4",
+    title: "Dark Matter",
+    icon: bgmap_darkmatter,
+  },
+];
```
Comment (KaWaite) [inline plugin/web/components/sidebar/tabs/MapSettingTab.tsx:31]: These will need i18n support in the future, so please move these in to the component.

File: plugin/web/components/sidebar/tabs/MapSettingTab.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    },
+    100: {
+      style: {
+        color: "#f50",
+      },
+      label: <Text>performance</Text>,
+    },
+  };
+  console.log(mapBing);
+  return (
```
Comment (KaWaite) [inline plugin/web/components/sidebar/tabs/MapSettingTab.tsx:50]:
```suggestion

  return (
```

File: plugin/web/components/sidebar/tabs/MapSettingTab.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+`;
+const ImageButton = styled(Radio.Button)`
+  margin: 0px 0px 12px 12px;
+  border-radius: 4px;
+  background: #d1d1d1;
+  padding: 4px 8px;
+  width: 64px;
+  height: 64px;
+`;
+const RasterMapSlider = styled(Slider)`
```
Comment (KaWaite) [inline plugin/web/components/sidebar/tabs/MapSettingTab.tsx:132]:
```suggestion

export default memo(MapSettingTab);

const MapViewSection = styled(Row)`
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  padding: 0px;
  gap: 16px;
  width: 296px;
  height: 103px;
`;

const MapViewButton = styled(Button)`
  width: 91px;
  height: 29px;
  border-radius: 4px;
  border-color: #d1d1d1;
  background: #d1d1d1;
  margin: 0px 0px 6px 6px;
  padding: 4px 8px;
`;

const BaseMapSection = styled(Row)`
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  padding: 0px;
  gap: 8px;
  width: 326px;
  height: 86px;
`;

const ImageButton = styled(Radio.Button)`
  margin: 0px 0px 12px 12px;
  border-radius: 4px;
  background: #d1d1d1;
  padding: 4px 8px;
  width: 64px;
  height: 64px;
`;

const RasterMapSlider = styled(Slider)`
```

File: plugin/web/components/sidebar/tabs/MapSettingTab.tsx (hunk 1 above)
Comment (nina992) [reply plugin/web/components/sidebar/tabs/MapSettingTab.tsx]: f0981a019a1257559ab6b090ae8069822b8ac5a3

**Review by KaWaite — APPROVED** [review]

**Pull request merged**